File: Python/pythagoras.py
from math import sqrt
def pythagoras(a=0, b=0, c=0):
    if a == 0 and b != 0 and c!= 0:
        print("A = " + str(sqrt(c*c - b*b)))
    elif b == 0 and a != 0 and c != 0:
        print("B = " + str(sqrt(c*c - a*a)))
    elif c == 0 and a != 0 and b != 0:
        print("C = " + str(sqrt(a*a + b*b)))
    else:
        print("Der er sket en fejl")

while True:
    print("""------------Pythagoras-------------
        Skriv 0 for ukendt""")
    a = int(input("Indsæt a: "))
    b = int(input("Indsæt b: "))
    c = int(input("Indsæt c: "))
    pythagoras(a, b, c)

File: learnC#/pythoagoras/pythagoras.cs
using System;

namespace pythoagoras
{
    internal class Program
    {
        public static void Main(string[] args)
        {
            Console.WriteLine("------------Pythagoras-------------");
            Console.WriteLine("Skriv 0 for ukendt");
            Console.Write("Indsæt a: ");
            double a = Convert.ToDouble(Console.ReadLine());
            Console.Write("Indsæt b: ");
            double b = Convert.ToDouble(Console.ReadLine());
            Console.Write("Indsæt c: ");
            double c = Convert.ToDouble(Console.ReadLine());

            double num = pythagoras(a, b, c);
            Console.WriteLine($"Ukendt: {num}");
        }

        static double pythagoras(double a = 0, double b = 0, double c = 0)
        {
            if (a == 0 && b != 0 && c != 0)
            {
                return Math.Sqrt(c * c - b* b);
            }
            else if (a != 0 && b == 0 && c != 0)
            {
                return Math.Sqrt(c * c - a * a);
            }
            else if (a != 0 && b != 0 && c == 0)
            {
                return Math.Sqrt(a * a + b * b);
            }
            else
            {
                return 0;
            }
        }
        
    }
}